Directory.Build.targets

```diff
@@ -1,6 +1,6 @@
 <Project>
   <Choose>
-    <When Condition="'$(TargetFramework)' == 'uap10.0' or '$(TargetFramework)' == 'native'">
+    <When Condition="'$(TargetFramework)' == 'uap10.0' or '$(TargetFramework)' == 'uap10.0.16299' or '$(TargetFramework)' == 'native'">
```

Comment:
Should we remove `uap10.0` to remove confusion?

Reply:
The notification library still uses uap10.0 as the min version there is th1